generator/osm2meta.cpp

```diff
@@ -287,6 +287,19 @@ string MetadataTagProcessorImpl::ValidateAndFormat_wikipedia(string v) const
   return normalized;
 }
 
+string MetadataTagProcessorImpl::ValidateAndFormat_wikimedia_commons(string v) const
+{
+    if(strings::StartsWith(v, "File:") || strings::StartsWith(v, "Category:"))
+    {
+        return v;
+    }
+    else
+    {
+        LOG(LDEBUG, ("Invalid Wikimedia Commons tag value:", v));
+        return string();
+    }
```

Comment:
```suggestion
  if (strings::StartsWith(v, "File:") || strings::StartsWith(v, "Category:"))
  {
    return v;
  }
  else
  {
    LOG(LWARNING, ("Invalid Wikimedia Commons tag value:", v));
    return {};
  }
```

Reply:
Imho, keep LDEBUG like all other metadata values here. Will be tons of useless entries in generator log.

Reply:
Sounds good.

Incorrect tags are quite frequent in OSM, especially for a tag that's growing as fast as this.  This can generate a lot of log messages.